core/src/main/java/org/akaza/openclinica/bean/core/DataEntryStage.java

```diff
@@ -27,7 +27,7 @@
     public static final DataEntryStage LOCKED = new DataEntryStage(7, "locked", "locked");
 
     private static final DataEntryStage[] members =
-        { UNCOMPLETED, INITIAL_DATA_ENTRY, INITIAL_DATA_ENTRY_COMPLETE, DOUBLE_DATA_ENTRY, DOUBLE_DATA_ENTRY_COMPLETE, ADMINISTRATIVE_EDITING, LOCKED };
```

Comment:
Can we create a new list for this. This list might be used in many places and don't want to introduce issues around this at this point

Reply:
do you mean like this (commit 98d2af912253469af02ef08bc34c2054c2996c90
) sir?

Reply:
You got it